hsm/sdf/SDFCryptoProvider.h

```diff
@@ -67,8 +195,8 @@ class SDFCryptoProvider : public CryptoProvider
     /**
      * Return the instance
      */
-    static SDFCryptoProvider& GetInstance();
-    static SDFCryptoProvider& GetInstance(int sessionPoolSize);
+    static SDFCryptoProvider& GetInstance(const std::string& libPath = "/usr/lib64/libgmt0018.so");
```

Comment:
```suggestion
    static SDFCryptoProvider& GetInstance(const std::string& libPath = "/usr/local/lib/libgmt0018.so");
```

Reply:
has modified default path of lib file from "/usr/lib64/libgmt0018.so" to "/usr/local/lib/libgmt0018.so"